airbyte-ci/connectors/pipelines/pipelines/airbyte_ci/connectors/context.py

```diff
@@ -50,6 +50,7 @@ def __init__(
         ci_gcp_credentials: Optional[Secret] = None,
         ci_git_user: Optional[str] = None,
         ci_github_access_token: Optional[Secret] = None,
+        ci_github_maintenance_token: Optional[Secret] = None,
```

Comment:
I'd really love to use `ci_github_access_token` here, and make the extra effort of generating a token / PAT with the right level of permissions. I don't think there's a strong reason to differentiate both token logically in code.